internal/veracode-api-wrapper/updatemitigationinfo/README.md

```diff
@@ -0,0 +1,48 @@
+**Veracode API Wrapper - Update Mitigation Info**
+
+Calls the Veracode api to update the mitigation comment of a build id. This is effectively used in updating the comment 
+in veracode which is maintained in CSV files.
+
+**Prerequisite**
+
+Prerequesties specific to Veracode :  Veracode Subcription , User role with "Mitigation" and "Mitigation and Comments"
+Prerequesties specif to program    :  <a href="https://github.com/wso2/security-tools/blob/master/internal/veracode-api-wrapper/updatemitigationinfo/requirements.txt">requirements.txt</a>
+
+**Local Installation**
+
+`$ virtualenv veracodeApiWrapper<br/>`
```

Comment:
Use 3 back-ticks and group all the commands into a section where you can copy and paste into terminal. 
Assume that user is in the "veracode-api-wrapper/updatemitigationinfo" folder when you write the commands. (`cd` might be unnecessary)